docs/guide/backend-integration.md

```diff
@@ -100,11 +105,14 @@ If you need a custom integration, you can follow the steps in this guide to conf
    }
    ```
 
-   - The manifest has a `Record<name, chunk>` structure
-   - For entry or dynamic entry chunks, the key is the relative src path from project root.
-   - For non entry chunks, the key is the base name of the generated file prefixed with `_`.
-   - For the CSS file generated when [`build.cssCodeSplit`](/config/build-options.md#build-csscodesplit) is `false`, the key is `style.css`.
-   - Chunks will contain information on its static and dynamic imports (both are keys that map to the corresponding chunk in the manifest), and also its corresponding CSS and asset files (if any).
+   The manifest has a `Record<name, chunk>` structure where each chunk follows the [`ManifestChunk`](https://github.com/vitejs/vite/blob/main/packages/vite/src/node/plugins/manifest.ts#L17-L28) interface. Each entry in the manifest represents one of the following:
+   - **Entry chunks**: Generated from files specified in [`build.rollupOptions.input`](https://rollupjs.org/configuration-options/#input). These chunks have `isEntry: true` and their key is the relative src path from project root.
+   - **Dynamic entry chunks**: Generated from dynamic imports. These chunks have `isDynamicEntry: true` and their key is the relative src path from project root.
+   - **Non-entry chunks**: Generated when code is split into separate chunks. Their key is the base name of the generated file prefixed with `_`. Their file names are controlled by [`build.rollupOptions.output.chunkFileNames`](https://rollupjs.org/configuration-options/#output-chunkfilenames).
+   - **Asset chunks**: Generated from imported assets like images, fonts, or CSS files. Their key is the relative src path from project root (without `_` prefix). Asset file names are controlled by [`build.rollupOptions.output.assetFileNames`](https://rollupjs.org/configuration-options/#output-assetfilenames).
+   - **CSS files**: When [`build.cssCodeSplit`](/config/build-options.md#build-csscodesplit) is `false`, a single CSS file is generated with the key `style.css`.
```

Comment:
When `build.cssCodeSplit` is not `false`, the key is generated similar to JS chunks (i.e. entry chunks will not have `_` prefix and non-entry chunks will have `_` prefix).